templates/ckancredentials.yaml

```diff
@@ -6,7 +6,7 @@ type: Opaque
 stringData:
   sysadminPassword: {{ .Values.ckan.sysadminPassword }}
   psqlMasterPassword: {{ .Values.ckan.psql.masterPassword }}
-  ckanSqlAlchemyUrl: "postgresql://{{ .Values.ckan.db.ckanDbUser }}:{{ .Values.ckan.db.ckanDbPassword }}@{{ .Values.ckan.db.ckanDbUrl }}/{{ .Values.ckan.db.ckanDbName }}"
-  ckanDatastoreWriteUrl: "postgresql://{{ .Values.ckan.datastore.RwDbUser }}:{{ .Values.ckan.datastore.RwDbPassword }}@{{ .Values.ckan.datastore.RwDbUrl }}/{{ .Values.ckan.datastore.RwDbName }}"
-  ckanDatastoreReadUrl: "postgresql://{{ .Values.ckan.datastore.RoDbUser }}:{{ .Values.ckan.datastore.RoDbPassword }}@{{ .Values.ckan.datastore.RoDbUrl }}/{{ .Values.ckan.datastore.RoDbName }}"
+  ckanSqlAlchemyUrl: "postgresql://{{ .Values.ckan.db.ckanDbUser }}@{{ .Values.ckan.db.ckanDbUrl }}:{{ .Values.ckan.db.ckanDbPassword }}@{{ .Values.ckan.db.ckanDbUrl }}/{{ .Values.ckan.db.ckanDbName }}"
+  ckanDatastoreWriteUrl: "postgresql://{{ .Values.ckan.datastore.RwDbUser }}@{{ .Values.ckan.datastore.RwDbUrl }}:{{ .Values.ckan.datastore.RwDbPassword }}@{{ .Values.ckan.datastore.RwDbUrl }}/{{ .Values.ckan.datastore.RwDbName }}"
+  ckanDatastoreReadUrl: "postgresql://{{ .Values.ckan.datastore.RoDbUser }}@{{ .Values.ckan.datastore.RoDbUrl }}:{{ .Values.ckan.datastore.RoDbPassword }}@{{ .Values.ckan.datastore.RoDbUrl }}/{{ .Values.ckan.datastore.RoDbName }}"
```

Comment:
@blazhovsky Why are we manually consructing the SQL alchemy URL? Instead of just using it from the env var?

Reply:
Ignore this.